package.json

```diff
@@ -1,11 +1,12 @@
 {
   "version": "1.0.0-alpha.0",
-  "author": "Community for NL Design System",
+  "author": "Community for Logius Design System",
```

Comment:
Mijn eerste gedachte is 'Author: Logius'. Dat is iets minder ambigu.

Reply:
Goed punt. Doorgevoerd